FWCore/Integration/test/run_TestSwitchProducer.sh

```diff
@@ -38,6 +38,37 @@ pushd ${LOCAL_TMP_DIR}
   echo "Test provenance of reversely merged output (Task)"
   cmsRun ${LOCAL_TEST_DIR}/${test}ProvenanceAnalyzer_cfg.py testSwitchProducerTaskMerge2.root || die "cmsRun ${test}ProvenanceAnalyzer_cfg.py Task2" $?
 
+
+  echo "*************************************************"
```

Comment:
General for this file, would be good to state why you are running this with # threads > 1.